README.adoc

```diff
@@ -50,7 +50,7 @@ patterns provided include Service Discovery (Eureka).
 
 == Features
 
-* Service Discovery: Eureka instances can be registered and clients can discover the instances using Spring-managed beans
+* Service Registration: Eureka instances can be registered and clients can discover the instances using Spring-managed beans
```

Comment:
https://github.com/spring-cloud/spring-cloud-netflix/blob/5cc8194b533577286435f98d93c4490b38f65396/README.adoc?plain=1#L1-L5

Please edit src/main/asciidoc/README.adoc

Reply:
I would like to update the changes in the src/main/asciidoc/README.adoc